doc/userguide/basics.rst

```diff
@@ -0,0 +1,118 @@
+.. _userguide_basics:
+
+Basics of Using SaltProc
+========================
+
+Running a Model
+---------------
+SaltProc is built in Python, so rather than calling an executable to run
+SaltProc, you call Python and invoke the SaltProc module. You use the ``-i``
+flag to specify a path to the :ref:`saltproc_input`:
+
+.. code-block:: bash
+
+   python -m saltproc -i saltproc_input.json
```

Comment:
In the future, you can create a true shell command (e.g. "saltproc") which doesn't require the `python -m` prefix. You can do this by adding it to your setup.py file (see pygenesys for an example).